modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -213,41 +214,43 @@ compression parameters :
         return 0;
     }
 @endcode
- */
+@param filename Name of the file.
+@param img Image to be saved.
+@param params Format-specific save parameters encoded as pairs, see cv::ImwriteFlags
+paramId_1, paramValue_1, paramId_2, paramValue_2, ... .
+*/
 CV_EXPORTS_W bool imwrite( const String& filename, InputArray img,
               const std::vector<int>& params = std::vector<int>());
 
-/** @overload */
```

Comment:
This change is not needed, because doxygen sorts functions by itself before putting them in documentation. I'd suggest to split documentation as follows:
```
/** <same as current documentation> 
*/
CV_EXPORTS_W Mat imdecode( InputArray buf, int flags );

/** @overload
@param buf <no documentation>
@param flags < no documentation>
@param dst <parameter description>
*/
CV_EXPORTS Mat imdecode( InputArray buf, int flags, Mat* dst);
```